content/v2/zones.markdown

```diff
@@ -117,3 +117,48 @@ Responds with HTTP 200, renders the zone file.
 <%= pretty_print_fixture("/getZoneFile/success.http") %>
 ~~~
 
+
+## Get zone distribution {#get-zone-distribution}
```

Comment:
I would keep the feature in beta for at least a few months, so that we can have breaking changes. I would add a note at the beginning (warning) that this feature is currently considered beta and breaking changes may happen.